corehq/apps/couch_sql_migration/tinypanda.py

```diff
@@ -0,0 +1,125 @@
+import operator
+
+
+def identity(value):
+    return value
+
+
+class TinyPanda:
```

Comment:
This is a table data structure, right? Whence the funny name? The popular python library `pandas`?

Reply:
Correct. The [original script](https://gist.github.com/millerdev/79d34a26a2f325125a32c65dd5315c50) on which `couch_domain_report` is based on used pandas. I wrote this because I didn't want to add pandas as a dependency of HQ just for this one management command.